bindata/assets/kueue-operator/clusterroles/clusterrole-manager-role.yaml

```diff
@@ -17,6 +17,14 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
```

Comment:
Why would the operand need this?

We are creating the config map in the operator and I think we have those RBAC here.